[IMP] empty

arpi-odoo opened the pull request.

arpi-odoo committed:
[FIX] hr: new contract button invisible

arpi-odoo committed:
[IMP] hr_skills: cleanup certification reporting

arpi-odoo committed:
[FIX] hr_holidays: fix traceback leave type

arpi-odoo committed:
[FIX] hr: fix delete version

arpi-odoo committed:
[FIX] hr: fix various bugs

mepe-odoo committed:
[FIX] hr_holidays: fix accrual day selection

arpi-odoo committed:
[IMP] empty

robodoo commented:
This PR targets the un-managed branch odoo-dev/odoo:master-hr-onboarding, it needs to be retargeted before it can be merged.

Commit:
[FIX] hr: New Contract button Invisible on no contract

Steps to reproduce:
1. Open "Employees" App
2. Click on any employee
3. Click on the "Payroll" Tab
4. Make sure "contract" date is empty

Bug cause:
The "button_new_contract" widget in the employee_form view is always displayed because it does not define an invisible condition.

Solution:
Added an invisible condition to the button_new_contract widget so it is shown only when the contract start date is set.

Task: #5447352

Commit:
[FIX] hr_holidays: Fixed Traceback on Time Off Type Issue

Steps to reproduce:
1. Time Off >> Configuration >> Time Off Types >> Open any type, navigate to the Smart button Time Off >> Click on New >> Time Off Type

Leads to a traceback error.

Bug cause:
The op function called in _search_virtual_remaining_leaves method expects two values [ie. op(value1, value2)], but only one was passed.
This led to the traceback error.

Solution:
Passed a second parameter "value" in the op function call.
Also added unit test for search_virtual_remaining_leaves function

task-5449083
